src/certif/proof.ml

```diff
@@ -25,24 +25,18 @@ module HS = HStringSExpr
 module H = HString
 
 (* Hard coded options *)
-  
-let mpz_proofs = true
+
 let debug = List.mem "certif" (Flags.debug ())
 let compact = not debug
 let set_margin fmt = pp_set_margin fmt (if compact then max_int else 80)
 
-
-
 (* disable the preprocessor and tell cvc5 to dump proofs *)
 let cvc5_proof_args () =
-  let args = ["--lang=smt2"; "--simplification=none"; "--dump-proofs"; "--proof-format=lfsc"] in
+  let args = ["--lang=smt2"; "--simplification=none"; "--dump-proofs"; "--proof-format=lfsc"; "--proof-granularity=theory-rewrite"] in
```

Comment:
Argument `"--proof-granularity=theory-rewrite"` should not be here. It is added below if `Flags.Certif.smaller_holes ()` returns `true`.

Reply:
This appeared because I was trying to get fancy with git rebase. The best way to fix this is to get even fancier!